### DISCUSSION TITLE
CentOS, but is RHEL worth trying

### DISCUSSION THREAD

**Discussion by john3641** [tethysplatform/tethys#721] [Q&A] [answered]
Hello,
I am trying to decide between using Digital Ocean to host Tethys Platform (in which case I think I would specify Ubuntu 20.04 server), or someone else for essentially no money on RHEL 7.9, but an investment in getting permissions and other clearances from IT department.   I know Tethys Platform is compatible with CentOS, and CentOS is an open source version of Red Hat, but anyone ever try Tethys Platform on RHEL?  Or have any intuition about whether it might be worth trying?

**Comment by swainn** [comment, marked as answer]
Hi @john3641 thanks for posting and sorry for the delay. I usually try to be more prompt in responding to discussions. I seem to remember that the original servers we hosted the Tethys demo site on were Red Hat servers. Unfortunately, the demo site was taken down a few years ago, so I can't verify for sure. I imagine that RHEL will be very similar to CentOS, though. This article or similar articles may be helpful: [CentOS vs. RHEL | OpenLogic](https://www.openlogic.com/blog/centos-vs-redhat).

The challenge with both of those distributions is managing the Enterprise Linux features, which is an additional security layer on top of the normal Linux file permissions. If you aren't required to use Enterprise Linux, you can disable it to get around that, though you'll lose the security advantages that CentOS/RHEL tout. 

Aside from that, the other major difference is the package manager (yum instead of apt).

**Reply by john3641** [reply]
Thanks so much for your thorough answer, Swainn.  I really have no experience deploying anything onto a server, so with this complication of an additional security layer you mention, I think I'll start with Ubuntu server on Digital Ocean.  If I build up enough working knowledge I might eventually talk with my university's IT folks about deploying on the RHEL server.  Thanks again for your help.

**Reply by swainn** [reply]
@john3641 I failed to mention that we have an online course with video walk-throughs of deploying Tethys Platform on CentOS. Anyone can sign up and take this course here:

* [Tethys Platform Production Deployment](https://tethys.edunext.io/courses/course-v1:tethys+PRD-100+2020-OPEN/about)

I'd highly recommend taking this course before your begin.

**Reply by john3641** [reply]
Thanks, Swainn.  So, I just enrolled in that course.  Thanks for the advice.  It looks like I'll need to do some pre-reqs first, so it might be a while before I actually start.  I do have a related question that I'm having trouble finding the answer to, but I will ask as a new question.
